data/json/items/armor/ballistic_armor.json

```diff
@@ -765,22 +762,22 @@
       {
         "id": "security_armor_vest",
         "name": { "str": "security ballistic vest" },
-        "description": "Lightweight, black vest used by security that may get shot.  Armored panels give this vest solid defense against small caliber firearms.  Has \"SECURITY\" written on the front lapel and across the back, and a pocket for a notebook and pen.",
+        "description": "A lightweight black vest used by private security, covering the chest and most of the abdomen.  Ballistic plates sealed inside the fabric provide solid defense against shrapnel and small-caliber rounds.  \"SECURITY\" is emblazoned on the front lapel and across the back, and it features a pocket for a notebook and pen.",
```

Comment:
These don't have "Ballistic plates" or pouches for them. They instead have soft armor panels 
![image](https://user-images.githubusercontent.com/4514073/162962080-000dae0f-7a58-453d-b2bb-ac07a7636404.png)

if they could take hard plates they would be level 3 or level 4 protection.